docs/running-frontend-against-examples.md

```diff
@@ -26,22 +26,6 @@ To start an instance of the dummy content store, from this repo, run:
 $ bundle exec dummy_content_store
```

Comment:
This doesn't exist. I think it's actually just the `bundle exec ruby app.rb` command we've used in GOV.UK Docker. Even then, the guidance won't work if you're running with GOV.UK Docker. I'm not aware that anyone actually uses this functionality for local development, so one option would just be delete the last section.

Reply:
I've now removed this section 👍